### PR TITLE
add new config files from the new CIS Kubernetes Benchmark

### PR DESCRIPTION
**Pull request by yoavrotems**
there is a new update at CIS_Kubernetes_Benchmark_v1.4.0 for Kubernetes 1.13

### CODE REVIEW

File: cfg/1.13/master.yaml [hunk 1]
```diff
@@ -0,0 +1,1500 @@
+---
+controls:
+version: 1.11
```
Comment (lizrice) [inline cfg/1.13/master.yaml]: Change to 1.13 please

File: cfg/1.13/node.yaml [hunk 2]
```diff
@@ -0,0 +1,480 @@
+---
+controls:
+version: 1.11
```
Comment (lizrice) [inline cfg/1.13/node.yaml]: And here too

**Comment by ttousai** [comment]
@lizrice @yoavrotems please also consider the changes made in #228.

**Comment by lizrice** [comment]
That's a very good point. cfg/1.13/node.yaml will need to take the same changes that are going into cfg/1.11/node.yaml under #228

**Comment by lizrice** [comment]
This will fix #209

**Pull request merged**